hs_core/tests/api/rest/test_create_resource.py

```diff
@@ -55,6 +55,48 @@ def test_post_resource_get_sysmeta(self):
             self.assertEqual(response['Content-Type'], 'application/zip')
             self.assertGreater(int(response['Content-Length']), 0)
 
+    def test_post_resource_get_sysmeta(self):
+        rtype = 'GenericResource'
```

Comment:
@hyi: PS: I just found a bug in my merge due to these tests. They've already proven their worth:)